server/app/services/DateConverter.java

```diff
@@ -117,6 +119,14 @@ public String renderDate(Instant time) {
     return dateTime.format(DATE_TIME_FORMATTER_WITH_SLASH);
   }
 
+  /** Formats an {@link Instant} to a short date in the local time zone and preferred locale. */
```

Comment:
**nitpick:** It would be nice to have an example of a short date in the comment, like we have above at line 109.

Reply:
done